landing_page/index.html

```diff
@@ -0,0 +1,467 @@
+            fill="white" />
+          <path
+            d="M111.027 17.6318V27C111.027 29.2091 112.818 31 115.027 31H130.64C132.849 31 134.64 29.2091 134.64 27V13.7812"
+            stroke="#659FF5" stroke-width="2" />
+          <path d="M145.046 2.39787L146.495 3.39253L147.987 4.57566V2L145.046 2.39787Z" fill="#F5BB65" />
+          <path
+            d="M110 18.3125C113.637 13.8837 118.849 7.45299 121.296 4.43059C122.033 3.52016 123.388 3.43748 124.234 4.24812L133.388 13.0255C134.154 13.7597 135.36 13.7683 136.136 13.0451L146.495 3.39253M146.495 3.39253L145.046 2.39787L147.987 2V4.57566L146.495 3.39253Z"
+            stroke="#F5BB65" stroke-width="3" />
+        </g>
+      </g>
+      <defs>
+        <clipPath id="clip0">
+          <rect width="1151" height="38" fill="white" />
+        </clipPath>
+        <clipPath id="clip1">
+          <rect width="149" height="38" fill="white" />
+        </clipPath>
+      </defs>
+    </svg>
+
+    <div class="row" style="margin-top: 94px;">
```

Comment:
I'm thinking it may be a good idea to clean up some of the sizing and margins in this file. If they are standardized a bit (ex. margins using increments of 4px, etc) I think that'll help with the mobile responsiveness.

Reply:
Maybe I got a bit carried away with directly copying values from the figma 😆